### PR TITLE
sending the mailing will ask for a confirmation

### PR DESCRIPTION
**Pull request by bartj3**

### CODE REVIEW

**Comment by bartj3** [comment]
closing to see if i can readjust the other pull-request to include the correct branch (using hub)

File: app/views/refinery/mailchimp/admin/campaigns/send_options.html.erb [hunk 1]
```diff
@@ -26,7 +26,7 @@
   <p>This will send the campaign.</p>
   <%= form_tag refinery.send_now_mailchimp_admin_campaign_path(@campaign) do -%>
     <%= render :partial => "modal_fields" %>
-    <%= submit_tag "Send Now"%>
+    <%= submit_tag "Send Now", :confirm => "You're about to email all the subscribers on your list, are you sure?"%>
```
Comment (johanb) [inline app/views/refinery/mailchimp/admin/campaigns/send_options.html.erb:29]: This is not i18n friendly. Please update.

Comment (bartj3) [inline app/views/refinery/mailchimp/admin/campaigns/send_options.html.erb:29]: The complete file is not i18n friendly, there's a seperate issue for that

Comment (johanb) [inline app/views/refinery/mailchimp/admin/campaigns/send_options.html.erb:29]: Even so, new additions can be done properly right away, no ? ;)

Comment (bartj3) [inline app/views/refinery/mailchimp/admin/campaigns/send_options.html.erb:29]: meh, but it's adjusted now :) translations of entire file on it's way, tags in the files are finished, only the actual translations left :)

**Pull request closed without merging**